examples/calibrateTask.py

```diff
@@ -52,81 +58,97 @@ def loadData(pixelScale=1.0):
     calib.setExptime(1.0)
     exposure.setCalib(calib)
     # add a filter
-    filterName = "r"
-    afwImage.Filter.define(afwImage.FilterProperty(filterName, 600, True))
-    exposure.setFilter(afwImage.Filter(filterName))
+    afwImage.Filter.define(afwImage.FilterProperty(FilterName, 600, True))
+    exposure.setFilter(afwImage.Filter(FilterName))
     # and a trivial WCS (needed by MyAstrometryTask)
-    pixelScale /= 3600.0                # degrees per pixel
-    wcs = afwImage.makeWcs(afwCoord.Coord(afwGeom.PointD(15, 1)), afwGeom.PointD(0, 0),
-                           pixelScale, 0.0, 0.0, pixelScale)
+    pixelScale /= 3600.0  # degrees per pixel
+    wcs = afwImage.makeWcs(Coord(PointD(15, 1)), PointD(0, 0), pixelScale, 0.0, 0.0, pixelScale)
     exposure.setWcs(wcs)
 
     return exposure
 
-class MyAstrometryTask(AstrometryTask):
-    """An override for CalibrateTask's astrometry task"""
-    def __init__(self, *args, **kwargs):
-        super(MyAstrometryTask, self).__init__(*args, **kwargs)
+class MyAstrometryTask(pipeBase.Task):
+    """An override for CalibrateTask's astrometry task that fakes a solution"""
+    ConfigClass = Config
+    _defaultName = "astrometry"
+
+    def __init__(self, schema=None, **kwargs):
+        pipeBase.Task(**kwargs)
 
     def run(self, exposure, sourceCat):
-        """My run method that totally fakes the astrometric solution"""
+        """Fake an astrometric solution
 
-        filterName = exposure.getFilter().getName()
-        wcs = exposure.getWcs()
-        #
-        # Fake a reference catalogue by copying fluxes from the list of Sources
-        #
-        schema = afwTable.SimpleTable.makeMinimalSchema()
-        schema.addField(afwTable.Field[float]("{}_flux".format(filterName), "Reference flux"))
-        schema.addField(afwTable.Field[float]("photometric", "I am a reference star"))
-        refCat = afwTable.SimpleCatalog(schema)
+        Pretend the current solution is perfect
+        and make a reference catalog that matches the source catalog
+        """
+        return self.loadAndMatch(exposure=exposure, sourceCat=sourceCat)
 
-        for s in sourceCat:
-            m = refCat.addNew()
-            flux = 1e-3*s.getPsfFlux()*np.random.normal(1.0, 2e-2)
-            m.set("{}_flux".format(filterName), flux)
-            m.setCoord(wcs.pixelToSky(s.getCentroid()))
-
-        refCat.get("photometric")[:] = True
-        #
-        # Perform the "match"
-        #
-        matches = []
-        md = dafBase.PropertyList()
-        for m, s in zip(refCat, sourceCat):
-            matches.append(afwTable.ReferenceMatch(m, s, 0.0))
+    def loadAndMatch(self, exposure, sourceCat):
+        """!Fake loading and matching
+
+        Copy the source catalog to a reference catalog and producing a match list
+        """
+        wcs = exposure.getWcs()
+        refSchema = LoadReferenceObjectsTask.makeMinimalSchema(
+            filterNameList = [FilterName],
```

Comment:
Well, http://developer.lsst.io/en/latest/coding/python_style_guide.html#spaces-must-not-be-used-around-for-default-parameter indicates that in the function definition there should not be spaces so I'm not sure why we would want them when we call the function. I'm having trouble finding the bit in the style guide where we talk about kwargs on separate lines. We are in a bit of a limbo where life would be much easier if we defined our style guide relative to PEP8.